fabtests/common/shared.c

```diff
@@ -85,6 +85,12 @@ int ft_socket_pair[2];
 
 fi_addr_t remote_fi_addr = FI_ADDR_UNSPEC;
 char *buf, *tx_buf, *rx_buf;
+/*
+ * tx_msg_buf are used by ft_fill_buf() to stage data sent over wire,
+ * when tx_buf is on device memory.
+ */
+void *tx_msg_buf = NULL;
```

Comment:
A better variable name would eliminate the need for the comment.  More to the point, why is this declared as a global, rather than in the ft_fill_buf call where it's used?

Reply:
Allocating pinned memory is very expensive, so I moved the allocation to `ft_alloc_msgs`, so we do not need to allocate/free it repeatedly.